lib/uktt/cli.rb

```diff
@@ -154,13 +154,13 @@ def handle_class_options(options)
 
       def action
         if options[:goods]
-          return :goods_nomenclatures
+          :goods_nomenclatures
         elsif options[:note]
-          return :note
+          :note
         elsif options[:changes]
-          return :changes
+          :changes
         else
-          return :retrieve
+          :retrieve
```

Comment:
```suggestion
          return :goods_nomenclatures if options[:goods]
          return  :note if options[:note]
          return :changes if options[:changes]
          
          :retrieve
```

Reply:
I'm actually tempted to remove the CLI aspect of this gem :thinking:

Reply:
I'll do it in another PR

Reply:
Yep